server/dist/views/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Why do wee need this Users ?

Reply:
I used some of the express app generators to setup server.
The users module was added out of the box.
Its the only place where I missed to remove user related functionality.